tests/components/landisgyr_heat_meter/test_config_flow.py

```diff
@@ -29,7 +34,9 @@ class MockUltraheatRead:
     device_number: str
 
 
-@patch("homeassistant.components.landisgyr_heat_meter.config_flow.HeatMeterService")
+@patch(
+    "homeassistant.components.landisgyr_heat_meter.config_flow.ultraheat_api.HeatMeterService"
```

Comment:
This could be made a `CONSTANT` in this module, since it's repeated this many times and causes wrapping of the `@patch` call.

Another option might be to make this `@patch` call an autouse fixture for this module.

Reply:
Good idea, I'll make it a constant for now. I'll look into the autouse fixture later.